include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -5698,7 +5698,9 @@ namespace internal
 
   /**
    * This struct implements the action of the inverse @ref GlossMassMatrix "mass matrix" operation
-   * using an FEEvaluationData argument.
+   * with user-provided coefficients at quadrature points (in contrast to
+   * CellwiseInverseMassMatrixImplBasic, which implicitly uses `1/(|J|xW)' as
+   * coefficient), using an FEEvaluationData argument.
```

Comment:
I guess this is not true for this function:
```suggestion
   * coefficient).
```